README.md

```diff
@@ -0,0 +1,21 @@
+# mlflow-elasticsearchstore
+
+Mlflow plugin to use Elaticsearch as Tracking Store in Mlflow. To use this plugin you need a running instance of Elasticsearch 6.8.
+
+## Installation
+
+In a python environment :
+
+```bash
+$ git clone git clone https://github.com/criteo/mlflow-elasticsearchstore.git
+$ cd mlflow-elasticsearch
+$ pip install .
+```
+
+## Launch 
```

Comment:
I wouldn't name this part Launch since we don't actually launch the store, it's more a How to